components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.rdbms/src/main/java/org/wso2/carbon/cluster/coordinator/rdbms/RDBMSCommunicationBusContextImpl.java

```diff
@@ -117,6 +118,9 @@ private void createTables() {
             DatabaseMetaData metaData = connection.getMetaData();
             databaseType = metaData.getDatabaseProductName();
             databaseVersion = Integer.toString(metaData.getDatabaseMajorVersion());
+            if (databaseType.contains(DB2_DATABASE)) {
```

Comment:
@RAVEENSR why you had to add this code specific to here ? isnt it getting resolved from metaData.getDatabaseProductName(); ? How other db types resolved it

Reply:
DB2 product name changes with the specific versions(For an example DB2/LINUXX8664, DB2/NT). Hence, checks whether the product name contains "DB2".

Reply:
Shall we do that in getDatabaseProductName method. so it will be cleaner and future codes can use it without having to check it again ?

Reply:
Also please see whether you can check it case insensitive way also like containsIgnoreCase in org.apache.commons.lang3.StringUtils. AFAIK contains in case sensitive

Reply:
I don't think we can do it there. `getDatabaseProductName()` method comes from [DatabaseMetaData](https://docs.oracle.com/javase/8/docs/api/java/sql/DatabaseMetaData.html) interface in java.sql package. The implementation for this interface is given from the JDBC driver. So we don't have the control to address this in `getDatabaseProductName()` method.

Reply:
Ack.

Reply:
Check the case sensitivity please

Reply:
Fixed in https://github.com/wso2/carbon-coordination/pull/63